reflex/config.py

```diff
@@ -251,6 +255,19 @@ def recompute_field_values(self):
             except AttributeError:
                 pass
 
+    def get_event_namespace(self) -> Optional[str]:
+        """Get the websocket event namespace.
+
+        Returns:
+            The namespace for websocket.
+        """
+        if self.event_namespace:
+            return f'/{self.event_namespace.strip("/")}'
+
+        event_url = constants.Endpoint.EVENT.get_url()
+        match = re.match(r"^\w+://[^/]+(/.*)?$", event_url)
```

Comment:
suggest the use of stdlib `urllib.parse.urlsplit` here to improve readability:

```python
>>> import urllib.parse
>>> urllib.parse.urlsplit("https://foo.bar.com/api/2/event")
SplitResult(scheme='https', netloc='foo.bar.com', path='/api/2/event', query='', fragment='')
>>> urllib.parse.urlsplit("wss://example.com/this/is/my/cool/api/event")
SplitResult(scheme='wss', netloc='example.com', path='/this/is/my/cool/api/event', query='', fragment='')
```

```python
def get_event_namespace(self) -> Optional[str]:
    ...
    urlparts = urllib.parse.urlsplit(event_url)
    return urlparts.path
```

Reply:
ah nice!

Reply:
ah nice!